### ISSUE TITLE
rakkasjs 0.6.15 incompatible with latest @hattip/cookiejs 0.0.29 ?

### ISSUE THREAD

**Issue by rondonjon** [rakkasjs/rakkasjs#110]
After `npm i @hattip/cookie` I got the following Typescript errors in my rakkasjs project:

![image](https://user-images.githubusercontent.com/8258823/230089413-a0d52e9b-b83a-4639-b2be-07e6ad99f035.png)

Not sure if there was a breaking change and hattip isn't using semantic versioning?

I was able to resolve the problem by downgrading the module to 0.0.27 (the same version as in the rakkasjs `package.json`)

**Comment by cyco130** [comment]
You're right, we should document which versions of `@hattip/*` are compatible with which versions of Rakkas until hattip versions stabilize.

In the meantime, please check the dependencies of the rakkasjs package (e.g. with `grep node_modules/rakkasjs/package.json -e hattip`) and install the same version.
